src/ipc.cpp

```diff
@@ -15,8 +15,8 @@ using namespace std;
 #define IPC_CHANNEL_PREFIX "/tmp/polybar_mqueue."
 #endif
 
-void log(const string& msg) {
-  fprintf(stderr, "polybar-msg: %s\n", msg.c_str());
+void display(const string& msg) {
+  fprintf(stdout, "%s\n", msg.c_str());
```

Comment:
I think we should keep the `polybar-msg: ` prefix.

```suggestion
  fprintf(stdout, "polybar-msg: %s\n", msg.c_str());
```

Reply:
This was intentional, identifying the origin of a message only make sense when writing to `stderr`. This [blog post](https://www.jstorimer.com/blogs/workingwithcode/7766119-when-to-use-stderr-instead-of-stdout) is good summary of why and how `stdout` and `stderr` are meant to be used.

If you insist I'll apply your suggestion though.

Reply:
Interesting. Though it doesn't say anything about `stdout` not needing to identify itself. But I see your point. I originally thought about us possibly adding more functionality to `polybar-msg` where polybar actually answers to certain commands and then we would actually need to identify the origin. But chances are that we're not even going to do anything like that ever, so I'll merge it like this.